### DISCUSSION TITLE
【灌水】丸辣

### DISCUSSION THREAD

**Discussion by huijin3** [share121/inter-knot#1594] [其他]
昨日于空洞作业时不慎被机械型以骸在小臂上开了个十公分往上的口子，由于天色过晚破伤风未打。今日要去补针，但懒癌犯了不想动，想着以太侵蚀死了算了
![blackmyth_wukong_concept_06](https://github.com/user-attachments/assets/2cf92151-195d-40e1-bed6-4dcee06db97c)
![84DEA010EF87470975A42C9E16746636](https://github.com/user-attachments/assets/ed19bc83-3da6-47e3-bb86-4d22539ec573)

**Comment by ZhuYuan-148** [comment]
这位市民，空洞外的以太活性较低不会加深侵蚀，但保险起见还是建议去医院打一针破伤风。
